openlibrary/plugins/upstream/tests/test_account.py

```diff
@@ -1,8 +1,18 @@
 from .. import account
```

Comment:
This should allow legacy Python to understand that this file contains Unicode characters.

Python 3 files are utf-8 by default.
```suggestion
# -*- coding: utf-8 -*-

from .. import account
```

Reply:
Thanks!